webapp/src/components/Navbar/Navbar.css

```diff
@@ -112,6 +112,47 @@
   display: none;
 }
 
+.Navbar .hamburger {
+  color: var(--primary);
+  margin-right: 0.5em;
+}
+
+.Navbar .hamburger-wrapper {
+  font-family: var(--font-family);
+  text-transform: uppercase;
+  font-weight: bold;
+  color: white;
+  display: flex;
+  flex-flow: row nowrap;
+  align-items: center;
+  cursor: pointer;
+  padding-bottom: 5px;
+}
+
+.Navbar .sidebar {
+  background: linear-gradient(#1f2333, #0d0f17) !important;
+  z-index: 2000 !important;
+  box-shadow: 0px 0px 0px 0px black !important;
+  transition: box-shadow 0.3s ease-in-out;
+}
+
+.Navbar .sidebar.visible {
+  background: linear-gradient(#1f2333, #0d0f17) !important;
+  z-index: 2000 !important;
+  box-shadow: 0px 0px 1500px 30px black !important;
+}
+
+.Navbar .menu.vertical .item {
+  font-size: 2em !important;
+  text-transform: none !important;
+  background-color: transparent !important;
+}
+
+.Navbar .activity-badge-mobile {
+  background: var(--primary) !important;
+  margin-top: 5px !important;
```

Comment:
I'm a bit scared at the amount of `!important` rules we're adding. Some of these can be safely removed because the selector has already a higher specificity than semantic's.

Do you think the `!important` are safer? I feel they're fine but dangerous for future changes, for example if this get's extracted into a lib and some project needs to do a tiny change, being able to overwrite a rule via specificity is key.

Reply:
Those `!important` are important :P I mean, if I remove those the `<Label>` is not aligned and it's rendered in gray instead of red. Maybe this can be addressed with a better specificity, but we cannot just remove them.